en/release-notes/upgrade-guide-v3.0.md

```diff
@@ -11,37 +11,31 @@ In order to ensure the smooth transition of an existing __Cocos Creator 2.4__ pr
 
-Users only need to click **File -> Import -> CocosCreator 2D Project** in the main menu, and a window for importing the plug-in will appear.
+Developers only need to click **File -> Import -> Cocos Creator 2D Project** in the main menu, and a window for importing the plug-in will appear.
 
-![image](https://user-images.githubusercontent.com/1503156/100599538-20b8c880-333b-11eb-9831-bf176730b777.png)
-![image](https://user-images.githubusercontent.com/1503156/100599556-26161300-333b-11eb-8b85-31e144300f73.png)
-![image](https://user-images.githubusercontent.com/1503156/100599545-23b3b900-333b-11eb-844a-876ad09fe7c6.png)
+![image](import-menu.png)
 
-Then click the button under the left picture and select the root directory of the __Cocos Creator__ 2D project. The plug-in will automatically traverse all the resources in the project and present it to the user. Users can check the resources to be imported by themselves, and click the import button in the right picture after selection. The import is complete.
+Next, click the button in the left picture below and select the root directory of the __Cocos Creator 2D__ project. The plug-in will automatically traverse all the resources in the project and present it to the developer. Developers can check the resources to be imported by themselves, and click the __Import__ button in the right picture below after selection. The import is complete.
 
-If an existing project needs to be upgraded for special reasons, and you encounter technical or workload difficulties, you can also contact [slackmoehrle@cocos2d-x.org](mailto:slackmoehrle@cocos2d-x.org) for assistance!
+![image](import.png)
+
+If an existing project needs to be upgraded under special circumstances, and technical or workload difficulties are encountered, please contact [slackmoehrle@cocos2d-x.org](mailto:Slackmoehrle@cocos.com) for assistance!
 
 ## Old version developers quickly get started
 
 ### Engine API upgrade
 
-#### Component class name change (for v1.2 users)
-
-In order to comply with the __Cocos Creator v2.x__ API specification, __Cocos Creator 3.0 Preview__ discarded the naming method of component class names containing Component suffix, and made automatic data upgrade and code compatibility.
+#### Asset loading
 
-However, it is recommended that developers still search for all uses of similar naming methods in the code and change to the class name without the Component suffix as soon as possible. You can use the following regular expressions to perform a global search (open case sensitivity and regular matching):
-
-```
-([A-Z]\w+)Component
-```
+The API for Cocos Creator 3.0 asset loading is consistent with v2.4, please refer to the [Asset Manager Overview](../asset/asset-manager.md).
 
-#### UI related interfaces on the obsolete node (for v2.x users)
+#### UI related interfaces on the obsolete node
 
 The abandoned interfaces are as follows:
```

Comment:
abandoned
-->
should we use discontinued or something else?

Reply:
deprecated?

Reply:
deprecated is great!